haproxy/tox.ini

```
@@ -3,6 +3,7 @@ minversion = 2.0
 basepython = py38
 envlist =
     py{27,38}-{16,17,18,20,21}
+    py{27,38}-{20,21}-new
```

Should we prefer a prefix for the legacy impl instead? ("New" won't stay relevant for long)

```suggestion
    py{27,38}-{16,17,18,20,21}-legacy
    py{27,38}-{20,21}
```